poker-pals/src/components/pokerComponents/ReportPocket.vue

```diff
@@ -147,27 +218,50 @@ export default {
     props: [
         "report_OffenderName",
         "report_OffenderMessageId",
+        "showForm",
+        "submittedSuccessfully"
     ],
     data(){
         return {
             selected: '',
-            message: '',
+            message: ''
         }
     },
     methods:{
-        cancelReport(){
-            this.$parent.cancelReport();
+        closeReport(){
+            this.$parent.closeReport();
         },
-        submitReport(){
-            if(this.selected === ''){
-                alert('please select reason'); // later use alternative option to indicate this
+        submitReport() {
+            // Check that all fields were filled 
+            if(this.selected === '' || this.message === ''){
+                this.setSelectionWarningVisibility(this.selected);
+                this.setReasonWarningVisibility(this.message);
             }
-            else{
-                // send off report to the parent
+            else {
+                // Send a filled report to the parent for submission to server
                 this.$parent.submitReport(this.selected, this.message, this.report_OffenderName, this.report_OffenderMessageId);
```

Comment:
After successfully submitting a report and closing the report screen. If you go to submit another report, are these warnings still showing?

Reply:
Nope.  Likely due to the use of v-if in the rendering.  It's a true re-render as opposed to just toggling the css display value, so it's getting a fresh copy of this component every time someone goes to file a report.